spec/integ/matrix-client-methods.spec.js

```diff
@@ -396,6 +395,236 @@ describe("MatrixClient", function() {
     });
 
     describe("partitionThreadedEvents", function() {
+
+        beforeEach(() => {
```

Comment:
I really hate beforeEach etc. - could we turn each variable into a function that returns a MatrixEvent instead?

(Reasoning: https://www.artificialworlds.net/blog/2013/03/27/setup-and-teardown-considered-harmful/ )